paymentsheet/src/main/java/com/stripe/android/paymentsheet/repositories/CustomerApiRepository.kt

```diff
@@ -151,4 +158,10 @@ internal class CustomerApiRepository @Inject constructor(
         ).onFailure {
             logger.error("Failed to update payment method $paymentMethodId.", it)
         }
+
+    private fun ErrorReporter.report(errorEvent: ErrorReporter.ErrorEvent, stripeException: StripeException) {
```

Comment:
Probably worth making this `internal`, and moving it to the same file as `EventReporter`.

Reply:
err, I guess it has to be public with an `@RestrictTo`.